.gitignore

```diff
@@ -37,3 +37,9 @@ artifacts-zk
 cache-zk
 deployments-zk
 deployments
+
+generated
```

Comment:
what's the difference between generated and artifacts?

Reply:
not sure what you mean,
generated/ is auto created by wagmi so i ignore it